### PR TITLE
Always clean before build.dev and add FORCE_TSLINT_EMIT_ERROR

### PR DESCRIPTION
**Pull request by karlhaas**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,8 +36,8 @@
     "start.deving": "gulp start.deving --color",
     "tasks.list": "gulp --tasks-simple --color",
     "test": "gulp test --color",
-    "e2e.ci": "gulp build.prod --color && gulp build.js.e2e --color && gulp e2e --color",
-    "tests.all": "npm test && npm run e2e.ci",
+    "e2e.ci": "gulp build.prod --color && gulp build.e2e --color && gulp e2e --color",
```
Comment (mgechev) [inline package.json]: The task name for building the project for e2e seems misspelled.

File: tools/config/seed.config.ts [hunk 2]
```diff
@@ -301,6 +301,12 @@ export class SeedConfig {
   ENABLE_SCSS = ['true', '1'].indexOf(`${process.env.ENABLE_SCSS}`.toLowerCase()) !== -1 || argv['scss'] || false;
 
   /**
+   * Enable tslint emit error by setting env variable FORCE_TSLINT_EMIT_ERROR
+   * @type {boolean}
+   */
+  FORCE_TSLINT_EMIT_ERROR = process.env.FORCE_TSLINT_EMIT_ERROR ? true : false;
```
Comment (mgechev) [inline tools/config/seed.config.ts]: Probably could be changed to simply `!!process.env.FORCE_TSLINT_EMIT_ERROR`.

File: package.json (hunk 1 above)
Comment (karlhaas) [inline package.json]: build.e2e runs clean.e2e as well. I think we had the cause that a test case has been renamed and build.js.e2e does not clean the old transpiled files

File: tools/config/seed.config.ts (hunk 2 above)
Comment (karlhaas) [inline tools/config/seed.config.ts]: changed

**Comment by mgechev** [comment]
@karlhaas the build is also failing https://ci.appveyor.com/project/mgechev/angular2-seed/build/2157.

**Comment by mgechev** [comment]
Thanks for the PR!

**Pull request merged**